src/main/java/com/jagex/runescape/Game.java

```diff
@@ -1917,6 +1917,8 @@ else if (playerRights == 1)
     }
 
     private DataInputStream openJaggrabStream(String request) throws IOException {
+        useJaggrab = Configuration.JAGGRAB_ENABLED;
```

Comment:
This seems to have been getting reset in places? So I make sure it's always set to the config value here...